src/main/java/it/gov/pagopa/payhub/auth/service/exchange/ValidateExternalTokenService.java

```diff
@@ -50,6 +51,9 @@ public void validateClient(String clientId) {
     }
 
     private void validateProtocolConfiguration(String grantType, String subjectTokenType, String scope) {
+        if (subjectTokenType == null) {
+            throw new IllegalArgumentException("subjectTokenType is mandatory with token-exchange grant type");
```

Comment:
don't throw a generic exception! this exception will be catched here:
it.gov.pagopa.payhub.auth.exception.AuthExceptionHandler
where it will be converted into an HttpStatus and possibly to a different DTO

I would use InvalidExchangeRequestException as an example, in order to reuse it and to not to create a new exception (to test)